### PR TITLE
Feature/invitation modal

### PR DESCRIPTION
**Pull request by satbir121**
1. Light and dark mode added in Invitation Modal.
2. Used component system.
3. `Create new org and share with a user` test cases fixed.

### CODE REVIEW

File: app/client/src/components/designSystems/appsmith/CopyToClipBoard.tsx [hunk 1]
```diff
@@ -1,36 +1,17 @@
...
-    background-color: #f6f7f8;
-    font-size: 14px;
-    &.bp3-button {
-      padding: 0px 0px;
-    }
+  div {
+    flex-basis: 82%;
+  }
+  a {
+    flex-basis: 18%;
```
Comment (satbir121) [inline app/client/src/components/designSystems/appsmith/CopyToClipBoard.tsx:14]: Use `calc(100% - 300px)`

File: app/client/src/components/editorComponents/form/FormDialogComponent.tsx [hunk 2]
```diff
@@ -5,10 +5,34 @@ import { isPermitted } from "pages/Applications/permissionHelpers";
...
+    background: ${props => props.theme.colors.modal.bg};
+    width: 640px;
+
     & .bp3-dialog-header {
-      padding: ${props => props.theme.spaces[4]}px
-        ${props => props.theme.spaces[4]}px;
+      padding: ${props => props.theme.spaces[4]}px;
+      background: ${props => props.theme.colors.modal.bg};
+      box-shadow: none;
+      .bp3-icon {
```
Comment (satbir121) [inline app/client/src/components/editorComponents/form/FormDialogComponent.tsx:17]: Use blueprint classes

File: app/client/src/components/ads/EditableText.tsx [hunk 3]
```diff
@@ -140,22 +140,23 @@ export const EditableText = (props: EditableTextProps) => {
...
   useEffect(() => {
     setValue(props.defaultValue);
     setIsEditing(!!props.isEditingDefault);
-  }, [props.defaultValue, props.isEditingDefault]);
+  }, [props]);
 
   useEffect(() => {
     if (props.forceDefault === true) setValue(props.defaultValue);
-  }, [props.forceDefault, props.defaultValue]);
+  }, [props]);
```
Comment (satbir121) [inline app/client/src/components/ads/EditableText.tsx:159]: try using `const {forceDefault, defaultValue} = props;` on top.

File: app/client/src/components/editorComponents/TagInputComponent.tsx [hunk 4]
```diff
@@ -1,23 +1,35 @@
...
 import styled from "styled-components";
 import { TagInput } from "@blueprintjs/core";
-import {
-  Intent,
-  IntentColors,
-  getColorWithOpacity,
-} from "constants/DefaultTheme";
+import { Intent } from "constants/DefaultTheme";
 const TagInputWrapper = styled.div<{ intent?: Intent }>`
+  margin-right: 8px;
```
Comment (satbir121) [inline app/client/src/components/editorComponents/TagInputComponent.tsx:6]: Check if this used elsewhere and move this into ads.

File: app/client/src/components/editorComponents/TagInputComponent.tsx [hunk 5]
```diff
@@ -1,23 +1,35 @@
...
+      background-color: ${props => props.theme.colors.tagInput.bg};
+      min-height: 38px;
+      border: 1px solid ${props => props.theme.colors.tagInput.bg};
+      border-radius: 0px;
+    }
+    .bp3-tag-input.bp3-active {
+      border: 1px solid ${props => props.theme.colors.info.main};
+      box-shadow: ${props => props.theme.colors.tagInput.shadow};
+    }
+    .bp3-input-ghost {
```
Comment (satbir121) [inline app/client/src/components/editorComponents/TagInputComponent.tsx:19]: Use blueprint Classes

File: app/client/src/components/stories/Dropdown.stories.tsx [hunk 6]
```diff
@@ -19,17 +19,14 @@ export const Text = () => (
         {
           id: "111abc",
           value: text("1st Option", "First option"),
-          onSelect: action("selected-option"),
```
Comment (satbir121) [inline app/client/src/components/stories/Dropdown.stories.tsx:22]: Revert

**Comment by mohanarpit** [comment]
@all-contributors Please add @satbir121 for their contributions to code, tests and ideas.

**Comment by mohanarpit** [comment]
@all-contributors Please add @satbir121 for their contributions to code, tests and ideas.

**Review by devrk96 — APPROVED** [review]

**Pull request merged**